tests/integration/test_e2e_workflows.py

```
@@ -0,0 +1,412 @@
+activate_to = ".env.production"
+''')
+
+        # Create base .env
+        env_file = work_dir / ".env"
+        env_file.write_text("APP_NAME=myapp\nDEBUG=false\n")
+
+        env = os.environ.copy()
+        env["PYTHONPATH"] = integration_pythonpath
+
+        # Test that profile-related commands don't crash
+        result = subprocess.run(
```

Ruff (0.14.10)

`subprocess` call: check for execution of untrusted input

(S603)